homeassistant/helpers/translation.py

```diff
@@ -388,31 +392,15 @@ async def async_get_translations(
     """
     if integrations is None and config_flow:
         components = (await async_get_config_flows(hass)) - hass.config.components
+    elif integrations is not None:
+        components = set(integrations)
```

Comment:
Why is this case removed?

Reply:
Its moved to line 398. I think its just the formatting??